### PR TITLE
2.0.0-dev.1.5

### PR DESCRIPTION
**Pull request by lexaknyazev**

### CODE REVIEW

File: tool/npm_template/index.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ exports.validateString = (json, options) => validator.validateString(json, optio
  @typedef {Object} ValidationOptions
  @property {string} uri - Absolute or relative asset URI that will be copied to validation report.
  @property {ExternalResourceFunction} externalResourceFunction - Function for loading external resources. If omitted, external resources are not validated.
+ @property {boolean} ignoreAccessorsData - Skip reading accessors data.
```
Comment (emackey) [inline tool/npm_template/index.js]: I have a personal preference against negative-sense Booleans like this, where `false` means "please validate" and `true` means "don't bother."  I've got stories of years worth of user confusion over flag names like this from other software.

Can we name this something with a positive sense, like `validateAccessorsData` with a default value of `true`?

Comment (donmccurdy) [reply tool/npm_template/index.js]: How about `includeAccessorData`?  (singular "accessor" sounds slightly more natural to me)

Comment (lexaknyazev) [reply tool/npm_template/index.js]: No strong opinion here, `include` sounds like `copy` to me. I'd prefer sticking to `validate`.

**Comment by emackey** [comment]
`2.0.0-dev.1.5` published to npm.

**Pull request merged**